packages/gatsby-theme-carbon/src/components/ImageCard/image-card.scss

```
@@ -38,13 +38,17 @@
   z-index: 1;
 }
 
+.#{$prefix}--image-card__title--has-subtitle {
```

Wrather than making a new class and handling this with JavaScript, could we try using a sibling selector and the already existing classes?

```suggestion
.#{$prefix}--image-card__subtitle ~ .#{$prefix}--image-card__title {
```